### PR TITLE
initial halfar petrinet model

### PR DESCRIPTION
**Pull request by danbryce**
This model involves discretizing the halfar model in one dimension to five spatial coordinates.  The slides at: 

https://docs.google.com/presentation/d/1DCHGtl40xfgfXiK__ZhdFo0SHN_i0HaD27fqRla8b_0/edit#slide=id.p

describe the model in more detail.

### CODE REVIEW

**Comment by bgyori** [comment]
Wow, Halfar model as Petri net was not on my bingo card for this hackathon but I really like it!

**Comment by danbryce** [comment]
I'm working on a generator for these models that controls the discretization, but I'm not sure if this is a good repo to hold it.  I'll update this instance in the future to point to the generator script in the funman repo.

**Comment by bgyori** [comment]
I think the JSON example is definitely a good fit here but the code should probably be somewhere else.

File: petrinet/examples/halfar.json [hunk 1]
```diff
@@ -0,0 +1,258 @@
+{
+  "header": {
+    "name": "Halfar Model",
+    "schema": "https://raw.githubusercontent.com/DARPA-ASKEM/Model-Representations/petrinet_v0.1/petrinet/petrinet_schema.json",
```
Comment (bgyori) [inline petrinet/examples/halfar.json:4]: Should this be referring to the latest schema version: `[petrinet_v0.6](https://github.com/DARPA-ASKEM/Model-Representations/releases/tag/petrinet_v0.6)`?

Comment (danbryce) [reply petrinet/examples/halfar.json:4]: Yes, will fix.

File: petrinet/examples/halfar.json [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        }
+      }
+    ]
+  },
+  "semantics": {
+    "ode": {
+      "rates": [
+        {
+          "target": "w_n_0",
+          "expression": "-1*gamma*(h_1-h_0)^3*h_0^5"
```
Comment (bgyori) [inline petrinet/examples/halfar.json:175]: One more small issue I noticed is that downstream (at least our processing pipeline) `**` is expected for exponentiation, not `^`. Could that be changed?

Comment (danbryce) [reply petrinet/examples/halfar.json:175]: Yes, no problem.

**Review by bgyori — APPROVED** [review]
Thanks!

**Pull request merged**